loader_hub/file/base.py

```diff
@@ -20,6 +20,8 @@
     ".epub": "EpubReader",
     ".md": "MarkdownReader",
     ".mbox": "MboxReader",
+    ".eml": "UnstructuredReader",
+    ".html": "UnstructuredReader",
```

Comment:
i'm fine with landing this but this is also outside the scope of this PR right?

Reply:
I wanted to make sure that if html files are downloaded, they are parsed correctly since `SimpleDirectoryReader` is used

Reply:
ah right